zerver/tornado/event_queue.py

```diff
@@ -1202,12 +1199,7 @@ def maybe_enqueue_notifications_for_message_update(
         # model.
         return
 
-    # We can have newly mentioned people in an updated message.
-    mentioned = user_profile_id in mention_user_ids
-
-    online_push_enabled = user_profile_id in online_push_user_ids
-
-    idle = (user_profile_id in presence_idle_user_ids) or receiver_is_off_zulip(user_profile_id)
+    idle = idle or receiver_is_off_zulip(user_profile_id)
```

Comment:
This is confusing code; we should call the parameter passed in `presence_idle`.  I'll just change that before merging.